posts/2022-09-22-real-time-everything.md

```diff
@@ -0,0 +1,249 @@
+---
+title: 'Real-time “everything” with operational and analytics data'
+description: 'Breakdown the walls between operational and analytics for real-time data sharing, starting with a shared love language to translate between JSON and SQL'
+date: '2022-09-22'
+categories:
+- architecture
+- concepts
+- tutorials
+- product
+- datamodeling
+authors:
+  - Francine Klein
+---
+## Real-time "everything" with operational and analytics data

```

Comment:
```suggestion
```

Reply:
Don't need to restate the title here because it'll come from the title snippet above.